### PR TITLE
Reformat Iris default keymap

### PR DESCRIPTION
**Pull request by nooges**

### CODE REVIEW

File: keyboards/iris/keymaps/default/keymap.c [hunk 1]
```diff
@@ -16,81 +14,63 @@ enum custom_keycodes {
...
+  //├────────┼────────┼────────┼────────┼────────┼────────┼────────┐        ┌────────┼────────┼────────┼────────┼────────┼────────┼────────┤
+     _______, _______, _______, _______, _______, _______, _______,          _______, _______, _______, _______, _______, _______, _______,
+  //└────────┴────────┴────────┴───┬────┴───┬────┴───┬────┴───┬────┘        └───┬────┴───┬────┴───┬────┴───┬────┴────────┴────────┴────────┘
+                                    _______, _______, _______,                   _______, _______, _______
+                                // └────────┴────────┴────────┘                 └────────┴────────┴────────┘
   )
-
 };
 
 void persistent_default_layer_set(uint16_t default_layer) {
```
Comment (drashna) [inline keyboards/iris/keymaps/default/keymap.c]: If you're reformatting/refactoring the code, would you mind removing this function (`persistent_default_layer_set`) and replace all references to it with `set_single_persistent_default_layer` instead?  (it should be called like `set_single_persistent_default_layer(_QWERTY)`)?

File: keyboards/iris/keymaps/default/keymap.c [hunk 2]
```diff
@@ -16,93 +14,70 @@ enum custom_keycodes {
...
-  eeconfig_update_default_layer(default_layer);
-  default_layer_set(default_layer);
-}
-
 bool process_record_user(uint16_t keycode, keyrecord_t *record) {
   switch (keycode) {
     case QWERTY:
       if (record->event.pressed) {
-        persistent_default_layer_set(1UL<<_QWERTY);
+        set_single_persistent_default_layer(1UL << _QWERTY);
```
Comment (drashna) [inline keyboards/iris/keymaps/default/keymap.c]: The `1UL <<` part isn't necessary.  The function includes that automatically, so all you need is the layer number/"name".

**Comment by drashna** [comment]
Awesome, thanks!

**Comment by pvinis** [comment]
@nooges besides changing the small function and changing the layout_kc to layout, which i understand, why the rest of the changes? why remove the rgb buttons etc?

**Comment by Xyverz** [comment]
@pvinis Probably to make it easier for him. I agree that the RGB controls should have been kept on the Adjust layer, and that those RGB changes should have been made to his own keymap, but I'm not the person who created the Iris keyboard, so I'm not sure I have much of a say in the matter.

I do like the change to LAYOUT instead of LAYOUT_kc, since it makes it easier for me to simply copy-paste my other keymaps into my Iris layout. I can always move the RGB controls on my own keymap.c file to the ADJUST layer if I want.

**Comment by pvinis** [comment]
yeah I was mostly asking for the rbg example essentially getting removed from the default. I guess people who want to find how to do it will find how to do it, same as with every other feature..

**Comment by nooges** [comment]
The RGB functions are still in the default, just moved to the Raise layer now to consolidate layers.

**Pull request merged**